### ISSUE TITLE
Outdated Node Requirement

### ISSUE THREAD

**Issue by JasonGore** [Financial-Times/disable-tree-shaking-for-chunk-plugin#10]
I can't even install this package as it has a dated and very strict Node requirement on `12.x`, which is fairly dated. LTS is v16, which should be the oldest version required. Can this be made looser, like `> 12.x`?

```
[3/5] Fetching packages...
error disable-tree-shaking-for-chunk-plugin@2.0.0: The engine "node" is incompatible with this module. Expected version "12.x". Got "16.14.2"
error Found incompatible module.
```

**Comment by ivomurrell** [comment]
Fixed in #8

**Comment by zimplexing** [comment]
v2.0.0 version also has same issue
